docs/reference/ilm/policy-definitions.asciidoc

```diff
@@ -0,0 +1,293 @@
+[role="xpack"]
+[testenv="basic"]
+[[index-lifecycle-management]]
+= Policy Phases and Actions
+
+There are four stages in the index lifecycle:
+
+* Hot--the index is actively being updated and queried.
+* Warm--the index is no longer being updated, but is still being queried.
+* Cold--the index is no longer being updated and is seldom queried. The
+information still needs to be searchable, but it's okay if those queries are
+slower.
+* Delete--the index is no longer needed and can safely be deleted.
+
+Each of these stages is called a "phase". A policy does not need to configure
+each phase for an index. For example, one index may only go from the hot phase to
+the delete phase, while another may run through all phases. Depending on which
```

Comment:
I think this is a bit confusing as is because it might seem like the same policy will be executed differently on different indexes. Maybe instead we should say something like the following?
```
For example, one policy may define only the hot phase and
the delete phase, while another may define all four phases.
```

Reply:
+1